charts/common/values.yaml

```diff
@@ -0,0 +1,600 @@
+
+      # podSecurityContext holds pod-level security attributes and common container settings.
+      # Some fields are also present in container.securityContext. Field values of containerSecurityContext take precedence over field values of podSecurityContext.
+      # ref: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#podsecuritycontext-v1-core
+      podSecurityContext:
+        enabled: false
+        # fsGroup: 2001
+        # runAsGroup: 1001
+        # runAsNonRoot: true
+        # runAsUser: 1001
+        # supplementalGroups: []
+
+      nodeSelector: {}
+      tolerations: []
+      affinity: {}
+
+      # volumes is a list of volumes to be created as secret or configMap
+      volumes: []
+        # name of the volume
+        #- name: generic-name
+        # type can either be "secret", "configMap" or "persistentVolumeClaim"
+        #  type: "secret"
+        # START ONLY FOR SECRET or CONFIGMAP
+        # filePath is optional for specifying a filePath in the helm chart where the file is located.
+        #  filePath: "files/xy.yml"
+        # defaultMode mode bits to use on created files by default. Must be a value between 0 and 0777. Comment out for not setting any and using default (0644)
+        #  defaultMode: 0644
+        # END ONLY FOR SECRET or CONFIGMAP
+        # START ONLY FOR PERSISTENTVOLUMECLAIM
+        # claimNameSuffix has to be the same as .pvcs.[].name
+        #  claimNameSuffix: "name-of-pvc"
+        # readOnly will force the ReadOnly setting in VolumeMounts. Comment out for using default (false)
+        #  readOnly: true
+        # END ONLY FOR PERSISTENTVOLUMECLAIM
+      # end pod
+
+      # start containers
+      # dictionary of containers in a pod
+      containers: {}
+        #container-1:
+          # image is a dictionary for specifying properties about the container image
+          #image:
+            # repository is the path to the image repository including container image name
+            #repository: registry-group.mgmtbi.ch/docker_base/nginx
+            # tag specifies the imagetag of the container to use. Set to "" for using .Values.defaultTag
+            #tag: "1.16"
+            # pullPolicy can be set to "Always". Comment out for using default ("IfNotPresent")
+            #pullPolicy: Always
+
+          # command Entrypoint array. Not executed within a shell. The docker image's ENTRYPOINT is used if this is not provided.
+          #command: []
+
+          # args Arguments to the entrypoint. The docker image's CMD is used if this is not provided.
+          #args: []
+
+          # containerSecurityContext holds security configuration that will be applied to a container.
+          # Some fields are present in both containerSecurityContext and podSecurityContext. When both are set, the values in containerSecurityContext take precedence.
+          # ref: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.23/#securitycontext-v1-core
```

Comment:
change to general link: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/

Reply:
done